### PR TITLE
Fix typo on landing page

### PR DESCRIPTION
**Pull request by marjev**
@mjfrey @vkaracic 

I fixed the typo found on the Landing Page. 
[SOL-1663](https://openedx.atlassian.net/browse/SOL-1663) is the JIRA ticket.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @marjev! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=582

**Comment by vkaracic** [comment]
:+1:

File: ecommerce/templates/coupons/_offer_discount.html [hunk 1]
```diff
@@ -57,7 +57,7 @@
                     <ul>
                         <li>{% trans "Certificate of achievement to show future employers to demonstrate skills and capabilities" %}</li>
                         <li>{% trans "Institution stamped and signed to verify your achievement" %}</li>
-                        <li>{% trans "Give yourself an additional incentive to complete to complete the course. You're 12x more likely to complete the course if you verify" %}</li>
+                        <li>{% trans "Give yourself an additional incentive to complete the course. You're 12x more likely to complete the course if you verify" %}</li>
```
Comment (clintonb) [inline ecommerce/templates/coupons/_offer_discount.html]: 12 times

File: ecommerce/templates/coupons/_offer_discount.html [hunk 2]
```diff
@@ -57,7 +57,7 @@
                     <ul>
                         <li>{% trans "Certificate of achievement to show future employers to demonstrate skills and capabilities" %}</li>
```
Comment (clintonb) [inline ecommerce/templates/coupons/_offer_discount.html]: Where did this list come from? Did @srpearce review? They all seem a bit off to me in terms of grammar and punctuation.

Comment (marjev) [inline ecommerce/templates/coupons/_offer_discount.html]: https://openedx.atlassian.net/browse/SOL-1626

**Comment by pbaruah** [comment]
This text was provided by the marketing team and we had asked @srpearce to review and approve.

File: ecommerce/templates/coupons/_offer_discount.html (hunk 2 above)
Comment (srpearce) [inline ecommerce/templates/coupons/_offer_discount.html]: I'm looking at the screen shot in SOL-1626 now. I've seen the text in the top bar and in the right pane before, but I've never seen "Why Verified Certificate" or the items under that heading—if I was tagged in a PR about that, I missed it. Hold tight for a minute and I'll provide updated text.

Comment (srpearce) [inline ecommerce/templates/coupons/_offer_discount.html]: These are my recommendations:

Why Should I Pursue a Verified Certificate?
- A verified certificate demonstrates to future employers that you’ve mastered the course material.
- The certificate is officially signed and stamped by the institution that offers the course. 
- You’re twelve times more likely to complete the course if you’re working toward a verified certificate.

**Comment by pbaruah** [comment]
:+1:

File: ecommerce/templates/coupons/_offer_discount.html [hunk 3]
```diff
@@ -55,9 +55,9 @@
                 </div>
                 <div class="col-md-5">
                     <ul>
-                        <li>{% trans "Certificate of achievement to show future employers to demonstrate skills and capabilities" %}</li>
-                        <li>{% trans "Institution stamped and signed to verify your achievement" %}</li>
-                        <li>{% trans "Give yourself an additional incentive to complete to complete the course. You're 12x more likely to complete the course if you verify" %}</li>
+                        <li>{% trans "A verified certificate demonstrates to future employers that you’ve mastered the course material." %}</li>
```
Comment (clintonb) [inline ecommerce/templates/coupons/_offer_discount.html:58]: Use a single quote rather than a smart quote for all of these.

**Pull request merged**